dev/com.ibm.ws.session/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -133,4 +133,7 @@ advanced.name=Advanced
 advanced.description=The default values of these options work for most environments.
 
 extraProperties.name=Additional properties
-extraProperties.description=Additional properties related to session management.
\ No newline at end of file
+extraProperties.description=Additional properties related to session management.
+
+modify.active.count.on.invalidated.session.name=Modify active count on invalidated session
+modify.active.count.on.invalidated.session.description=Httpsession activeCount metric may become inaccurate when the session is accessed by multiple applications, set the property to "false" to resolve the issue.
```

Comment:
Suggested changes:
- Change "may" to "might"
- Break the text into 2 sentences. Thus, change  "applications, set" to "applications. Set"